boards/opentitan/README.md

```diff
@@ -18,11 +18,11 @@ for more details.
 
 Programming
 -----------
+The latest supported version (commit SHA) of OpenTitan is specified in the [OPENTITAN_SUPPORTED_SHA](https://github.com/tock/tock/blob/master/boards/opentitan/earlgrey-cw310/Makefile) make variable found:
+>  boards/opentitan/earlgrey-cw310/Makefile
 
-Tock on OpenTitan requires
-lowRISC/opentitan@217a0168ba118503c166a9587819e3811eeb0c0c. In
-general it is recommended that users start with the specified commit as newer
-versions have not been tested.
+In *general* it is recommended that users start with the specified commit as newer
+versions **have not been** tested.
```

Comment:
This should say that `OPENTITAN_SUPPORTED_SHA` specifies this commit

Reply:
Fixed!